robotoff/insights/question.py

```diff
@@ -164,6 +170,47 @@ def get_source_image_url(barcode: str) -> Optional[str]:
 
         return None
 
+    @staticmethod
+    def generate_selected_images(images: JSONType, barcode: str) -> JSONType:
+        selected_images = {}
+        images_data = {}
+        front = {}
+        small = {}
+        thumb = {}
+        display = {}
+
+        splitted_barcode = split_barcode(barcode)  # splitting the barcode
+
+        for k in images.items():
+            key = k[0]
+            if key.startswith("front_"):  # to add support for all languages
+                language = key.split("_")[1]  # get language name
+                revision_id = images[key]["rev"]  # get revision_id for all languages
+
+                image_url = (
+                    IMAGE_SUB_DOMAIN
```

Comment:
You can you f-strings here, it would make the string more readable
And there is a `generate_image_url` in `off.py` that does exactly this